File: JavaScript/oop_project.js
import BankAccount from './BankAccount.js'
import InterestRewardsAcct from './InterestRewardsAcct.js';
import SavingsAcct from './SavingsAcct.js';

const Jared = new BankAccount(1000, "Jared");
Jared.getBalance();
Jared.deposit(100);
Jared.withdraw(10000);
const Nick = new BankAccount(1000, "Nick");
Jared.transfer(100, Nick);

const Zak = new InterestRewardsAcct(1000, "Zak");
Zak.getBalance();
Zak.deposit(100);

const Sara = new SavingsAcct(1000, "Sara");
Sara.withdraw(100);

File: Java/oop_project.java
/**
 * Test
 */
public class oop_project {
    public static void main(String[] args) {
        BankAccount Jared = new BankAccount(1200, "Jared");
        Jared.getBalance();
        Jared.deposit(1200);
        Jared.withdraw(10000);

        BankAccount Zak = new BankAccount(10, "Zak");
        Jared.transfer(100, Zak);

        InterestRewardsAcct Frank = new InterestRewardsAcct(1000, "Frank");
        Frank.getBalance();
        Frank.deposit(100);

        SavingsAcct Sebastian = new SavingsAcct(1000, "Sebastian");
        Sebastian.withdraw(100);
    }
}